src/components/services-view/services-view.html

```diff
@@ -1,8 +1,17 @@
 
 <div class="services-view">
 
-  <overlay-panel show-panel="$ctrl.orderingPanelvisible" show-close="true" handle-close="$ctrl.closeOrderingPanel">
-    <order-service service-class="$ctrl.serviceToOrder" handle-close="$ctrl.closeOrderingPanel"></order-service>
+  <overlay-panel show-panel="$ctrl.orderingPanelVisible" show-close="true" handle-close="$ctrl.closeOrderingPanel">
+    <order-service
+        ng-if="$ctrl.selectedServiceClass"
+        service-class="$ctrl.selectedServiceClass"
+        handle-close="$ctrl.closeOrderingPanel">
+    </order-service>
+    <builder-dialog
```

Comment:
<builder-dialog> isn't very descriptive.  The component name should give an indication on what the component does, similar to '<order-service>'.  How about '<create-builder>' or '<configure-builder>'?